### DISCUSSION TITLE
[Feature] Allow albums to be removed on webui when empty.

### DISCUSSION THREAD

**Discussion by Dental1347** [immich-app/immich#2280] [Feature Request]
### The feature

I assume this wouldn't qualify as a bug, hence why I report it as a feature request.
There are two different components to this post which I would like to address:

1. When creating an album under albums > create album. It will by default contain no photos and it has the default name of 'Untitled'. when you then click the arrow in the top left to navigate back the album has been created. It does what the button says, it creates the album. But there is no use in having an empty album if it is still named untitled and contains no photos. My suggestion would be to remove the album if the name hasn't been changed and it contains no images. 
![image](https://user-images.githubusercontent.com/44779473/232825867-314dd2a9-57d4-4b4b-867d-ebfceb312028.png) 
![image](https://user-images.githubusercontent.com/44779473/232826168-e450390c-5f38-49fa-8389-e1cc9cb20f39.png)

2. Make it so albums can be removed when no images are within the album. Currently, you need to upload at least one image or video to be able to remove the album. This feels unnecessary and is not user friendly. Please see the attached images.
![image](https://user-images.githubusercontent.com/44779473/232826796-4e620bcb-2cd8-4ad2-a79d-337141fd6b72.png)
![image](https://user-images.githubusercontent.com/44779473/232826841-f7cc2ea0-1b4a-40b4-9e53-bad894a3879c.png)

3. Bit related, but also not, when holding an album it will show the following: 
![image](https://user-images.githubusercontent.com/44779473/232826653-ab404ccd-8cd7-4fd3-8c3c-2c117e05938f.png)
This feels unintended, as the album is already part of immich and is not an external file.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by alextran1502** [comment]
Ah this is a bug. If the album is empty and has a title as "Untitled" it should be removed upon navigating back to the album page. I will take a look at this.

**Reply by Dental1347** [reply]
Thanks for the fast response @alextran1502
